Makefile

```diff
@@ -52,7 +52,7 @@ cmd/util/util:
 .PHONY: unittest-main
 unittest-main:
 	# test all packages with Relic library enabled
-	go test -coverprofile=$(COVER_PROFILE) -covermode=atomic $(if $(RACE_DETECTOR),-race,) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) --tags relic $(GO_TEST_PACKAGES)
+	go test -v -coverprofile=$(COVER_PROFILE) -covermode=atomic $(if $(RACE_DETECTOR),-race,) $(if $(JSON_OUTPUT),-json,) $(if $(NUM_RUNS),-count $(NUM_RUNS),) --tags relic $(GO_TEST_PACKAGES)
```

Comment:
did you mean to enable verbose logging here? I think that will get very noisy

Reply:
Yes, I enabled it because we have had issues in CI of test failures but without additional logging so it was hard to tell what was wrong. This is mostly for CI test failure diagnosis.

Reply:
ok. another option is the add a `VERBOSE` flag like in the `make -e VERBOSE test` case to enable it specifically in CI.

but in either case we can just adjust it later if it becomes a problem